package.json

```diff
@@ -33,9 +33,10 @@
     "sync-externals": "npm run-script sync-schema && npm run-script sync-colors",
     "sync-schema": "cd externals/metaphysics && git fetch && git checkout origin/master && yarn install && npm run dump-schema -- ../../data && rm -rf node_modules",
     "version": "yarn install && yarn bundle && pushd Example && bundle install && bundle exec pod install && popd && git add Pod/Assets Example/Podfile.lock",
+    "flow-remove-node-modules": "rm -rf node_modules/react-native/flow && find node_modules -type f -name '*.js' -print0 | xargs -0 grep -lE '@(no)?flow' | xargs -P $(expr $(sysctl -n hw.ncpu) + 1) -I '{}' ./node_modules/.bin/flow-remove-types --pretty --quiet '{}' --out-file '{}' || true",
```

Comment:
BOOM, wipe Flow from _all_ of `node_modules`. The two Babel parser plugins can’t coexist, so this was the simplest solution to make the source compatible.